js/navbar.js

```diff
@@ -6,7 +6,8 @@ window.addEventListener('DOMContentLoaded', (event) => {
     checkURL === productionURL
       ? '23c78f66ab7964e5ef97'
       : 'c4a84431feaf604e89d1';
-  userLoginEl.href = `https://github.com/login/oauth/authorize?client_id=${clientId}`;
+
+  userLoginEl.href = `https://github.com/login/oauth/authorize?client_id=${clientId}&state=${productionURL}`;
```

Comment:
Instead of productionURL use checkURL. 
To make sure even the pages are covered with this feature.

Reply:
Sure.

Reply:
Made suggested changes. Please check.